eldritch/encounters.py

```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
   roll3 = events.Gain(char, {"clues": 3})
   cond = events.Conditional(char, check, "successes", {0: roll0, 1: roll1, 3: roll3})
   return events.Sequence([check, cond], char)
+def Asylum2(char):
+  check = events.Check(char, "speed", -1)
+  item = events.Draw(char, "unique", 1)
+  move = events.ForceMovement(char, "Downtown")
+  cond = events.Conditional(char, check, "successes", {0: move, 1: item})
```

Comment:
You never use this conditional.

Reply:
yeah , realized that I didn't need to use a conditional for everything. removed